### ISSUE TITLE
Progressbar is not visible after upgrading to v6

### ISSUE THREAD

**Issue by prindacerk** [MurhafSousli/ngx-progressbar#272]
After upgrading ngx-progressbar to v6, the bar is not visible. I debugged and identified that the reason for this is because "-active" class is not getting added to the div with the class **ng-progress-bar**.

In v5, this gets added to the div during the display of the progress bar. In v6, there's a new property **active** in the div but it doesn't seem to work as the same as v5. When I force apply "-active" to the class of that div, the bar is visible once again.

Please advise how I can fix this.

#### Reproduction

Steps to reproduce:
1. Implement v5 of ngx-progressbar on an angular project
2. Upgrade project to v6 progressbar

#### Expected Behavior

Progressbar should work as it did before.

#### Actual Behavior

Progress is not visible.

#### Environment

- Angular: 8.2.13
- ngx-progressbar: 6.0.1
- Browser(s): Chrome
- Operating System (e.g. Windows, macOS, Ubuntu):  Windows

**Comment by MurhafSousli** [comment]
It looks fine in the [stackblitz](https://stackblitz.com/edit/ngx-progressbar), can you add a reproduction?

**Comment by prindacerk** [comment]
Hi,

I have identified the problem. It seems to be caused by the app id having "." in the name which is screwing up the style. My app id which I migrated has "." in it's name (like **ngx.progressbar.http**). In previous version, the styles were applied via class and not by the identifiers **_ngcontent-ngx.progressbar.http-c0**. So this was not an issue. In v6, it uses identifiers like this which seemed to have an issue with the app id that has fullstop in it.

When I changed the fullstop to -, it works fine.

Is this something that will be an issue in future versions? Should I be updating my app id without fullstops? Angular also seemed to restrict fullstops in app id.
